client-app/shared/catalog/components/product-price-block.vue

```diff
@@ -82,7 +119,15 @@ function addToList() {
   });
 }
 
+function getProductSocialShareUrl(urlTemplate: string, url: string): string {
```

Comment:
Why not to use stringFormat directly in the template? We have a proxy method with the same arguments for now.